3rdPartyCypress/cypress/integration/worldServiceCoUkRedirect.js

```diff
@@ -0,0 +1,39 @@
+import services from '../../../src/app/lib/config/services';
+
+Object.keys(services).forEach(index => {
+  const service = index;
+
+  const notWSServices = ['news', 'cymrufyw', 'naidheachdan', 'default']; // Not WS
+
+  if (notWSServices.includes(service)) {
+    return;
+  }
+  describe('WS Redirects:', () => {
+    it(`all routes should redirect for ${service}`, () => {
+      const urlsTotest = [
+        `https://www.bbc.co.uk/${service}`,
+        `https://www.bbc.co.uk/${service}/articles/a00000000o`,
+        `http://www.bbc.co.uk/${service}/articles/a00000000o`,
+        `https://www.bbc.co.uk/${service}/articles/a00000000o.amp`,
+        `http://www.bbc.co.uk/${service}/articles/a00000000o.amp`,
+      ];
+
+      urlsTotest.forEach(urlToTest => {
+        cy.request({
+          url: urlToTest,
+          followRedirect: false,
+        }).then(resp => {
+          expect(resp.status).to.eq(301);
+          // expect first slice to equal https://www.bbc.com/
+          expect(resp.redirectedToUrl.substring(0, 20)).to.eq(
+            'https://www.bbc.com/',
+          );
+          // expect second slice to equal whatever came in
+          expect(resp.redirectedToUrl.substring(20)).to.eq(
+            urlToTest.substring(22),
+          );
+        });
+      });
+    });
+  });
+});
```

Comment:
I'd suggest adding a script to the package.json to make code reviewing/testing locally a bit simpler. 
```
    "test:e2e:3rdParty:interactive": "npm run killApp && npm run build && run-p --race start cypress:3rdParty:interactive",
```

Reply:
That's a bit too nice, there's already a script, I'm trying to cut the number of scripts.

Reply:
I'll just set up an alias locally on my machine, then.